OHHTTPStubs/Sources/OHHTTPStubs.m

```diff
@@ -424,7 +424,28 @@ - (void)startLoading
                 // Notify if a redirection occurred
                 if (((responseStub.statusCode > 300) && (responseStub.statusCode < 400)) && redirectLocationURL)
                 {
-                    NSURLRequest* redirectRequest = [NSURLRequest requestWithURL:redirectLocationURL];
+                    NSURLRequest *redirectRequest = nil;
+                    NSMutableURLRequest *mReq = nil;
```

Comment:
This `= nil;` isn't needed as all path below will affect it a value anyway. (By not affecting it to `nil`, the compiler will warn us in case we later change the code and forget to affect a value in one of the code paths)